tox_win.ini

```diff
@@ -54,8 +54,8 @@ commands =
     python testing/rdifftest.py
 #    python testing/securitytest.py  # no os.getuid + backslash in path handlingon --server
 #    python testing/killtest.py  # cannot create symbolic link
-#    python testing/backuptest.py  # backslash in path handling on --server
-#    python testing/comparetest.py  # backslash in path handling on --server
+#    python testing/backuptest.py  # No module named rdiff_backup in server.py
```

Comment:
What is the meaning of this comment ?

Reply:
I think, Windows doesn't inherit the environment variables, so PYTHONPATH isn't properly set to point to the tox environment, so it doesn't find the rdiff-backup module.